src/components/GroupBy/GroupBy.vue

```diff
@@ -212,9 +219,9 @@ export default {
     }
   },
   computed: {
-    all() {
-      return all
-    },
+    // all() {
+    //   return all
+    // },
     ...mapGetters([
```

Comment:
Unsure what this did, so I disabled it with the plan to turn it back on  or remove it depending on what bug pop up. :)